Rarime/Code/Utilities/StringUtils.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import BigInt
+
+class StringUtils {
+    static func cropString(_ input: String, lenght: Int = 10) -> String {
+        return input.count > lenght * 2 ? "\(input.prefix(lenght))......\(input.suffix(lenght))" : input
```

Comment:
let's use maxLength instead of length and it should cover the whole string length, not a part